tests/erlang_tests/copy_terms0.erl

```diff
@@ -20,10 +20,10 @@
 
 -module(copy_terms0).
 
--export([start/0, loop/0, count_nestings/1]).
+-export([start/0, count_nestings/1]).
```

Comment:
I see there are a lot of changes from MFA -> fun in the tests.  I don’t have an issue with this per se, but just curious why.  (I see there are also some MFA tests below 👇🏽 , so we are getting coverage of both kinds.  Just curious why the change.

Reply:
I thought the usage of spawn/3 was historical, especially as spawn/3 and spawn/1 were not factorized. spawn/1 remove the need for exporting a function and is more natural. 

As you write, both are covered anyway.